lib/tasks/github_usage_tracker.rake

```diff
@@ -0,0 +1,6 @@
+namespace :github_usage_tracker do
+  desc "Record a current datapoint of GitHub API usage"
+  task :record_datapoint => :environment do
+    GithubUsageTracker.record_datapoint
+  end
```

Comment:
If we put it on the regular queue it could get backed up behind other work, but I would be fine with adding another queue for this kind of stuff.  Once it's on the queue, it should be picked up immediately.
```ruby
class MyWorker
  include Sidekiq::Worker
  include Sidetiq::Schedulable

  recurrence { minutely }
end
```